config/samples/otel.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinApp
+metadata:
+  name: otel-spinapp
+spec:
+  image: ghcr.io/spinkube/spin-operator/cpu-load-gen:20240311-163328-g1121986
+  executor: containerd-shim-spin
+  replicas: 1
+---
+apiVersion: core.spinoperator.dev/v1alpha1
+kind: SpinAppExecutor
+metadata:
+  name: containerd-shim-spin
+spec:
+  createDeployment: true
+  deploymentConfig:
+    runtimeClassName: wasmtime-spin-v2
+    installDefaultCACerts: true
+    otel:
+      endpoint: http://otel-collector-opentelemetry-collector.default.svc.cluster.local:4318
```

Comment:
How would the user have this running already? Can we document that somewhere? Or alternatively can we have this sample yaml also deploy the collector? That might be hard b/c I think you need to use helm.

Reply:
Yeah, I think we'd need a helm install. Can we document this using comments inside of the sample app, or is there a place we should document this?

Reply:
Comments inside the yaml seems fine to me. A good followup would be adding more explicit documentation to <https://spinkube.dev/docs>.